### PR TITLE
Separate modules

### PR DESCRIPTION
**Pull request by jayoshih**

### CODE REVIEW

File: ricecooker/classes/nodes.py [hunk 1]
```diff
@@ -172,7 +172,7 @@ class ContentNode(Node):
     """
     def __init__(self, *args, **kwargs):
         # Map parameters to model variables
-        assert isinstance(args[0], basestring), "id must be a string"
+        assert isinstance(args[0], str), "id must be a string"
```
Comment (jayoshih) [inline ricecooker/classes/nodes.py]: basestring outdated in python 3.x

File: ricecooker/managers/downloader.py [hunk 2]
```diff
@@ -0,0 +1,309 @@
+import hashlib
```
Comment (jayoshih) [inline ricecooker/managers/downloader.py]: Code copy/pasted from managers.py

File: ricecooker/managers/progress.py [hunk 3]
```diff
@@ -0,0 +1,236 @@
+import pickle
```
Comment (jayoshih) [inline ricecooker/managers/progress.py]: Code copy/pasted from managers.py

File: ricecooker/managers/tree.py [hunk 4]
```diff
@@ -0,0 +1,245 @@
+
```
Comment (jayoshih) [inline ricecooker/managers/tree.py]: Code copy/pasted from managers.py

File: ricecooker/commands.py [hunk 5]
```diff
@@ -6,8 +6,11 @@
 from . import config
 from .classes import nodes, questions
 from requests.exceptions import HTTPError
+from .managers.downloader import DownloadManager
```
Comment (aronasorman) [inline ricecooker/commands.py]: What was our reasoning for changing ricooker to use relative imports again?

**Pull request merged**